tests/unit/stream_alert_shared/test_lookup_tables.py

```diff
@@ -127,3 +128,21 @@ def test_load_lookup_tables(self, mock_logger):
 
         LookupTables.load_lookup_tables(self.config)
         mock_logger.assert_called()
+
+    @patch('logging.Logger.debug')
+    def test_load_lookup_tables_compresed(self, mock_logger):
+        """LookupTables - Load lookup table, compressed file"""
+        self.config['global']['infrastructure']['lookup_tables']['enabled'] = True
+        lookup_tables = LookupTables.load_lookup_tables(self.config)
+        # Replace one of the S3 objects with a compressed version
+        put_mock_s3_object(
+            'bucket_name',
+            'bar.json',
+            zlib.compress(json.dumps({'compressed_key': 'compressed_val'})),
+            self.region
+        )
+        result = lookup_tables.download_s3_objects()
+
+        assert_equal(result.get('bar'), {'compressed_key': 'compressed_val'})
+        assert_equal(result.get('foo'), {'bucket_name_key': 'foo_value'})
+        mock_logger.assert_any_call('Data in \'%s\' is not compressed', 'foo.json')
```

Comment:
The s3 object is compressed, in this try/except block, the debug message should not be triggered. 
```
+                 # The lookup data can optionally be compressed, so try to decompress
+                 # This will fall back and use the original data if decompression fails
+                try:
+                    data = zlib.decompress(data, 47)
+                except zlib.error:
+                    LOGGER.debug('Data in \'%s\' is not compressed', json_file)
```
Do I miss anything?

Reply:
@chunyong-lin there are two files (see your __init__) above - the `bar.json` is compress, the `foo.json` is not. The logger call we check for is `'Data in \'%s\' is not compressed', 'foo.json'`

Reply:
Thanks for the explanation! STAMP!